core/node/node_framework/src/implementations/resources/circuit_breakers.rs

```diff
@@ -0,0 +1,23 @@
+use std::sync::Arc;
+
+use zksync_circuit_breaker::CircuitBreakers;
+
+use crate::resource::Resource;
+
+#[derive(Debug, Clone, Default)]
+pub struct CircuitBreakersResource {
+    pub replication_lag_limit_sec: Option<u32>,
```

Comment:
I think this may be designed better. Currently, you implicitly rely on the fact that some layer would fetch and create this resource, set this value, and then you will use it to instantiate `ReplicationLagChecker`. This isn't the best design for a couple of reasons:
1. You create more implicit relationships between components, which may be confusing during debugging.
2. You have several components aware of this configuration, which breaks the point of framework where we want to encapsulate the initialization as much as possible.

To improve it, you may add this field to `Web3ServerOptionalConfig`, and take it from there. This way the `Web3ServerLayer` will be the only layer aware of `ReplicationLagChecker`.